mix.exs

```diff
@@ -74,8 +74,8 @@ defmodule Plausible.MixProject do
       {:envy, "~> 1.1.1"},
       {:ex_machina, "~> 2.3", only: [:dev, :test]},
       {:excoveralls, "~> 0.10", only: :test},
-      {:exvcr, "~> 0.11", only: :test},
-      {:finch, "~> 0.14.0", override: true},
+      {:exvcr, github: "ruslandoga/exvcr", branch: "use-jason-ruslan", only: :test},
+      {:finch, "~> 0.16.0"},
```

Comment:
I had to update Finch since exvcr Finch adapter defines request!/2,3 functions which are not present in finch 0.14